### ISSUE TITLE
javax.jms dependency not found

### ISSUE THREAD

**Issue by raboof** [akka/alpakka#2121]
In the jms subproject, we prepend the jboss resolver to the list of resolvers to make sure the jms dependency is fetched from the jboss repo's instead of from Maven Central, working around https://github.com/coursier/coursier/issues/200

This does not work reliably, because `csrRecursiveResolvers` does not reliably put the project resolvers before the resolvers from dependency projects (in this case: testkit). Because of that, on some systems, the testkit resolvers (which include Maven Central) would be put before the project resolvers (which start with the jboss repo), rendering the project unbuildable. Fix proposed upstream as https://github.com/sbt/sbt/pull/5413 , but let's work around in in Alpakka as well in the mean time.

**Comment by ennru** [comment]
Fixed with #2121

**Comment by seglo** [comment]
It may also be necessary to nuke local caches for coursier and any other local repositories that may be in your resolvers list.  

i.e.) I had to nuke my coursier and m2 local caches.

```
rm -r ~/.cache/coursier/v1/https/repo1.maven.org/maven2/javax/jms
rm -r ~/.m2/repository/javax/jms
```

**Comment by dwijnand** [comment]
> Fixed with #2121

#2122
